### PR TITLE
Get cpr details for private company

### PR DESCRIPTION
**Pull request by Araballi**

### CODE REVIEW

File: src/Kmd.Logic.Cpr.Client/Models/AddressInfromation.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the AddressInfromation class.
+        /// </summary>
+        public AddressInfromation(AddressPrivate address = default(AddressPrivate), string status = default(string), System.DateTime? efectFrom = default(System.DateTime?), bool? effectFromUncertaintyMarking = default(bool?))
+        {
+            Address = address;
+            Status = status;
+            EfectFrom = efectFrom;
```
Comment (MichalDudelo) [inline src/Kmd.Logic.Cpr.Client/Models/AddressInfromation.cs:29]: Please correct this to **effect**

File: src/Kmd.Logic.Cpr.Client/Models/CitizenPrivateResponse.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            LastName = lastName;
+            MiddleName = middleName;
+            LegalGuardian = legalGuardian;
+            DepartureAndEntry = departureAndEntry;
+            ContactAddress = contactAddress;
+            AddressInformation = addressInformation;
+            NameAndAddressProtection = nameAndAddressProtection;
+            Status = status;
+            StatusDate = statusDate;
+            StatusDateUncertaintyMarking = statusDateUncertaintyMarking;
```
Comment (MichalDudelo) [inline src/Kmd.Logic.Cpr.Client/Models/CitizenPrivateResponse.cs:38]: Please change to **statusDateUncertainityMarking**

**Comment by NakulSeerpu** [comment]
In read me document, CPR private section pls change the method name to the actual name.

**Review by NakulSeerpu — APPROVED** [review]
LGTM

**Comment by paimaam** [comment]
LGTM

**Comment by MichalDudelo** [comment]
LGTM

**Pull request merged**